src/FSAL/FSAL_CORTXFS/ds.c

```diff
@@ -217,7 +215,9 @@ kvsfs_ds_write(struct fsal_ds_handle *const ds_pub,
 
 
 	*written_length = amount_written;
-	*stability_got = stability_wanted;
+	
+	/* CORTX FS provides file sync stability */
```

Comment:
Please update the comment to reflect that CORTX FS only supports and provides file sync stability, rest are not supported.